### PR TITLE
FIX: Tooltip content scrolling

### PR DESCRIPTION
**Pull request by vepor**
https://take.ms/87h5Og

<br/><br/><br/><url>LiveURL: https://orbit-components-fix-tooltip-scrolling.surge.sh</url>

### CODE REVIEW

**Comment by tomashapl** [comment]
Could you please write some story for this case ? @vepor

**Comment by vepor** [comment]
I would not expose it, as it should not be used in this way - it's just legacy edge case. Also it won't be captured visually, as Tooltip is hidden by default. @tomashapl

**Comment by tomashapl** [comment]
> I would not expose it, as it should not be used in this way - it's just legacy edge case. Also it won't be captured visually, as Tooltip is hidden by default. @tomashapl

Understand, but it should be nice to have story with bigger content of tooltip to see if your fix works

**Comment by vepor** [comment]
Added @tomashapl

File: src/Tooltip/Tooltip.stories.js [hunk 1]
```diff
@@ -130,6 +130,24 @@ storiesOf("Tooltip", module)
         "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
     },
   )
+  .add(
```
Comment (tomashapl) [inline src/Tooltip/Tooltip.stories.js]: Please use there `lokiSkip` instead of `add` to prevent capturing

**Pull request merged**